products/developertoolset/_common/product.yml

```diff
@@ -1,7 +1,7 @@
 name: Red Hat Developer Toolset
 abbreviated_name: Developer Toolset
 product_logo: portal.png
-current_version: 3.1.0.GA
+current_version: 4.1
```

Comment:
Thank you. I suspected as much from looking at the log but wasn't very confident.

No, it shouldn't contain GA -- in fact, no RH product that I know of includes the letters "GA" in its version. The z-number is also used very rarely. I'm pushing a commit with quotes.